.github/workflows/test_cpp.yml

```diff
@@ -395,20 +430,23 @@ jobs:
             cache-prefix: windows-2019-cmake
             # windows-2019 has python3.7 installed, which is incompatible with the latest gcloud
             python-version: '3.9'
+            continuous-only: true
           - name: Windows CMake 32-bit
             os: windows-2022
             flags: >-
               -G Ninja -Dprotobuf_WITH_ZLIB=OFF -Dprotobuf_BUILD_CONFORMANCE=OFF
             vsversion: '2022'
             windows-arch: 'win32'
             cache-prefix: windows-2022-win32-cmake
+            continuous-only: true
           - name: Windows CMake Shared
             os: windows-2022
             flags: >-
               -G Ninja -Dprotobuf_WITH_ZLIB=OFF -Dprotobuf_BUILD_CONFORMANCE=OFF
               -Dprotobuf_BUILD_SHARED_LIBS=ON
             vsversion: '2022'
             cache-prefix: windows-2022-cmake
+            continuous-only: false
```

Comment:
We should probably remove all "continuous-only: false" just to minimize clutter

Reply:
Deleted here and in all other files.